lib/wanda/execution/evaluation.ex

```diff
@@ -82,10 +92,14 @@ defmodule Wanda.Execution.Evaluation do
         message: "Fact gathering ocurred during the execution"
```

Comment:
Not related to this PR but it might be worth a quick fix: is `error` missing in the sentence probably?

Reply:
fixed here ad731d5d300b6400664d930fd4fb83bfcdb5ec63